bucket/tortoisesvn.json

```diff
@@ -78,9 +78,9 @@
         ]
     ],
     "post_install": [
-        "$escdir = $dir -replace \"\\\\\",\"\\\\\"",
-        "(gc \"$dir\\tortoisesvn-install.reg.templ\") | % {$_ -replace \"%TSVNHOME%\",\"$escdir\"} | out-file -enc oem \"$dir\\tortoisesvn-install.reg\"",
-        "(gc \"$dir\\tortoisesvn-install-tools.reg.templ\") | % {$_ -replace \"%TSVNHOME%\",\"$escdir\"} | out-file -enc oem \"$dir\\tortoisesvn-install-tools.reg\"",
+        "$escdir = $dir -Replace \"\\\\\",\"\\\\\"",
+        "(Get-Content \"$dir\\tortoisesvn-install.reg.templ\") | % {$_ -Replace \"%TSVNHOME%\",\"$escdir\"} | Out-File -Encoding OEM \"$dir\\tortoisesvn-install.reg\"",
```

Comment:
- First are positional parameters then named
- DO not use `OEM`, Use `ASCII`
- `%` is alias of `ForEach-Object`
- Single quotes for `%TSVNHOME`

Reply:
Thanks for the details.